src/main/cli/commandLine.ts

```diff
@@ -6,16 +6,17 @@
 // ==LICENSE-END==
 
 import { readerActions } from "readium-desktop/common/redux/actions";
+import { PublicationView } from "readium-desktop/common/views/publication";
 import { diMainGet } from "readium-desktop/main/di";
 import { URL } from "url";
 import { isArray } from "util";
 
-export async function cli_(filePath: string) {
-    // import and read publication
-    const catalogService = diMainGet("catalog-service");
-    const publication = await catalogService.importFile(filePath);
-    const store = diMainGet("store");
+function openReader(publication: PublicationView | PublicationView[]) {
+    if (isArray(publication)) {
```

Comment:
Should be negative (`!isArray()`)

Reply:
no !

Reply:
Sorry, it was a "doh" moment on my part!
(I misread the code inside the `if` block)

Reply:
In fact your are right, i may add an openReader loop on each publicationView. But it's not very useful on our case.